### ISSUE TITLE
OverlayPanel: Missing aria-label on close icon

### ISSUE THREAD

**Issue by Rawi01** [primefaces/primefaces#5033]
## 1) Environment
- PrimeFaces version: **latest**
- Does it work on the newest released PrimeFaces version? Version? **No**
- Does it work on the newest sources in GitHub? (Build by source -> https://github.com/primefaces/primefaces/wiki/Building-From-Source) **No**
- Application server + version: **All**
- Affected browsers: **All**

## 2) Expected behavior

The close icon of the overlay panel sets an additional ARIA label (`aria-label="Close"`) to support assistive technology like screenreaders. Thats how it works for other overlay components too e.g. column toggler.

## 3) Actual behavior

Right now there is no label and the screenreader can not explain the user what this close link will do. 

The whole code gets rendered using jQuery here
https://github.com/primefaces/primefaces/blob/938d06d1b680612c55ab9beca3bc6ae128d7bea5/src/main/resources/META-INF/resources/primefaces/overlaypanel/overlaypanel.js#L20-L22

A possible fix for this one would be to add 
`.attr('aria-label', PrimeFaces.getAriaLabel('overlaypanel.CLOSE'))` 
to the construction code (thats how it works for column toggler right now). There also is a duplicate `href` attribute on the link that could be removed too :smile:

**Comment by melloware** [comment]
Removed the duplicate HREF and submitted PR.
